### PR TITLE
[Bugfix] Request WFS: Typename or feature id are mandatory

### PR DESCRIPTION
**Pull request by rldhont**
Lizmap Web Client is checking TYPENAME parameter, but TYPENAME is not mandatory if FEATUREID is defined.

* Funded by 3liz

### CODE REVIEW

**Comment by nboisteault** [comment]
Cypress fails in feature_toolbar test because `layerConfig.types` is `undefined`.

https://github.com/3liz/lizmap-web-client/blob/master/lizmap/www/assets/js/attributeTable.js#L1980

**Comment by rldhont** [comment]
@nboisteault cypress is green

File: lizmap/modules/lizmap/lib/Request/WFSRequest.php [hunk 1]
```diff
@@ -238,9 +274,9 @@ protected function process_getfeature()
         }
 
         // Get type name
-        $typename = $this->param('typename');
+        $typename = $this->requestedTypename();
         if (!$typename) {
-            \jMessage::add('TYPENAME is mandatory', 'RequestNotWellFormed');
+            \jMessage::add('TYPENAME or FEATUREID are mandatory', 'RequestNotWellFormed');
```
Comment (nboisteault) [inline lizmap/modules/lizmap/lib/Request/WFSRequest.php:279]:
```suggestion
            \jMessage::add('TYPENAME or FEATUREID is mandatory', 'RequestNotWellFormed');
```

File: tests/end2end/cypress/integration/requests-service-ghaction.js [hunk 2]
```diff
@@ -173,4 +220,22 @@ describe('Request service', function () {
             expect(resp.body).to.contain('WFS_Capabilities')
         })
     })
+
+    it('TYPENAME or FEATUREID are mandatory for WFS GetFeature request', function () {
```
Comment (nboisteault) [inline tests/end2end/cypress/integration/requests-service-ghaction.js:224]:
```suggestion
    it('TYPENAME or FEATUREID is mandatory for WFS GetFeature request', function () {
```

**Review by nboisteault — APPROVED** [review]
Not sure but if think `is` is correct not `are`.

**Pull request merged**